.github/workflows/_ci.yaml

```diff
@@ -7,26 +7,11 @@ on:
       ARCHITECTURE:
         type: string
         required: true
-      TRIAL_BRANCH:
-        type: string
-        description: 'Name of (branch|tag|sha) with bumped manifest and patches'
-        required: true
     outputs:
-      TAG_BASE:
-        description: "Tags of the base image built"
-        value: ${{ jobs.build-base.outputs.DOCKER_TAG }}
-      TAG_JAX:
-        description: "Tags of the JAX image built"
-        value: ${{ jobs.build-jax.outputs.DOCKER_TAG_FINAL }}
-      TAG_T5X:
-        description: "Tags of the T5X image built"
-        value: ${{ jobs.build-t5x.outputs.DOCKER_TAG_FINAL }}
-      TAG_PALLAS:
-        description: "Tags of the Pallas image built"
-        value: ${{ jobs.build-pallas.outputs.DOCKER_TAG_FINAL }}
-      TAG_PAX:
-        description: "Tags of the PAX image built"
-        value: ${{ jobs.build-pax.outputs.DOCKER_TAG_FINAL }}
+      ALL_TAGS:
```

Comment:
Since this is the only output related to tags, `ALL_` won't be necessary. How about just `CONTAINER_TAGS`?

Reply:
Done